### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by ndorrh**
**My Portfolio Desktop Version**
In this part of the project, I Optimize my portfolio for desktop view
Using media queries

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,15 +18,20 @@
...
     </header>
     <main class="main-content">
       <!--Headline Section-->
       <section class="head-line backgrnd-clr">
         <h2 class="greeting greeting-text headline-box">
-          Hey there. I’m <br />
-          Mariane
+          Hey there. I’m <br class="break" />
+          <span class="ma">Mariane</span>  <span class="Al">Alex</span>
           <span class="text-color">I’m a software developer</span>
```
Comment (tufoinnkuo10) [inline index.html:35]: - [ ] please kindly ensure that The Title heading is followed by 2 rows with 3 projects each (6 projects in total). Ensure they stay 3 projects in 2 rows on bigger screens.
Screenshot (322)_LI

File: index.html [hunk 2]
```diff
@@ -276,7 +282,7 @@ <h2 class="contact-message">
             ><img src="images/icons/angellist.svg" alt="angellist"
           /></a>
         </div>
-        <div>
+        <div class="email">
```
Comment (tufoinnkuo10) [inline index.html:285]: - [ ] great job. Everything looks fine and it's responsive. Please kindly use the right background image on the contact section.
Figma Version
Screenshot (324)_LI
your version

Screenshot (323)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @ndorrh 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ followed Figma design

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @ndorrh , I'm @MaganaAsiati , your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in future 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏

**Pull request merged**